### PR TITLE
ADD ✅ QUOTE ✅ Console logging 👍 

### PR DESCRIPTION
**Pull request by trstephen**
WIP for feedback.

Assumes the [fake quote server](https://github.com/DistributedDesigns/fake_quote_server) is running

### CODE REVIEW

File: accounts/accounts.go [hunk 1]
```diff
@@ -0,0 +1,54 @@
+package accounts
+
+import "errors"
+
+// Account : State of a particular account
```
Comment (trstephen) [inline accounts/accounts.go:5]: `gometalinter` complains if I don't have comments for exported stuff. It probably knows best.

File: accounts/accounts.go [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+type Account struct {
+	Balance float64
```
Comment (JakeCooper) [inline accounts/accounts.go:7]: Didn't we want cash-on-hand not represented by a float?

Comment (trstephen) [reply accounts/accounts.go:7]: Yeah, this is me cutting corners. ✂️ 

How does Go feel about operator overloading? First thing that comes to mind is to introduce a `Currency` type with `.dollars` and `.cents` with `+` and `-` overloaded to handle rollovers / unders between the two values.

Comment (JakeCooper) [reply accounts/accounts.go:7]: I don't think golang has operator overload. You'll have to make typeactions like 

```
func AddDollars(a account, value Currency) {
    a.dollars += value.dollars
    a.cents += value.cents
}
```

**Comment by trstephen** [comment]
Squashed and merged instead of using this as a feature branch.

**Pull request merged**